doc/src/stdlib/collections.md

```diff
@@ -133,6 +133,9 @@ Base.collect(::Type, ::Any)
 Base.issubset(::Any, ::Any)
 Base.filter
 Base.filter!
+# this also includes the docstring for replace(A, old, new, n)
```

Comment:
Can you post the exact code you tried? That could help somebody more familiar with these issues.